MacOSExample/metro.config.js

```diff
@@ -25,13 +33,6 @@ module.exports = {
       return acc;
     }, {}),
   },
-
-  transformer: {
```

Comment:
Why do we drop this setting? Is it included in the default config?

Reply:
Exactly, it is present in [default config](https://github.com/facebook/react-native/blob/5323221d1442d1573bc65daff618478cb6f056f0/packages/metro-config/index.js)